flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/stream/sql/join/JoinReorderITCase.java

```diff
@@ -42,6 +45,12 @@ public class JoinReorderITCase extends JoinReorderITCaseBase {
 
     private StreamExecutionEnvironment env;
 
+    @AfterEach
+    public void after() {
```

Comment:
This is the root cause? Do you reproduce failed test locally?

Reply:
> This is the root cause? Do you reproduce failed test locally?

This error can not be reproduce in local. I think the root cause is the `StreamTestSink.clear();` not be called after tests finished.